centos-ci/utils/get-vm-ip

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+
+import sys
+import time
+import libvirt
```

Comment:
Are we installing the libvirt python bindings?  Isn't this script `virsh domifaddr` anyways?  Or is CentOS 7 libvirt to old to know that?